src/containers/Subscription.js

```diff
@@ -51,7 +98,7 @@ export const Subscription = () => {
 
     fetchSegments();
```

Comment:
i have a feeling that because you are not ever checking whether segments were fetched this could lead to multiple calls ?

Reply:
Initially my thought was that I'd want it to re-render whenever "config" is changed, but I guess there really isn't any need for the process to be dynamic.
Updated to only run through if "segments" is null